web/src/main/java/org/openmrs/web/Listener.java

```diff
@@ -424,10 +429,31 @@ private void copyCustomizationIntoWebapp(ServletContext servletContext, Properti
 					}
 				}
 			}
-			
+			List<String> locales = Arrays.asList("", "_de", "_en_GB", "_es", "_fa", "_fr", "_hi", "_in_ID", "_it", "_kh",
```

Comment:
There's no easy way to load all files matching messages*.properties from classpath. He would have to use some classpath scanning utility which is slow. I think that hard coding this list is fine.